File: geoprocessamento-service/src/main/java/com/rodovia/geoprocessamento/DTO/Endereco.java
package com.rodovia.geoprocessamento.DTO;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
@JsonIgnoreProperties(ignoreUnknown = true)
public class Endereco {

    @JsonProperty("road")
    private String rua;

    @JsonProperty("suburb")
    private String bairro;

    @JsonProperty("town")
    private String cidade;

    @JsonProperty("state")
    private String estado;

    @JsonProperty("country")
    private String pais;

    @JsonProperty("postcode")
    private String cep;
    
}

File: webapp-geoprocessamento/src/app/geoprocessamento/interfaces/Endereco.ts
export interface Address {
    suburb: string;
    city: string;
    state: string;
    postcode: string;
    country: string;
    latitudeUsadoRequest: number;
    longitudeUsadoRequest: number;
}

export interface Endereco {
    address?: Address;
    display_name?: string;
    latitudeUsadoRequest?: number;
    longitudeUsadoRequest?: number;
    error?: boolean;
}